crypto/storage/storage.go

```diff
@@ -40,6 +40,8 @@ type Storage interface {
 	PrivateKeyExists(kid string) bool
 	// SavePrivateKey stores the key under the kid in the storage backend.
 	SavePrivateKey(kid string, key crypto.PrivateKey) error
+	// ListPrivateKeys returns the KIDs of the private keys that are present.
```

Comment:
Maybe mention not to use for features that also work in strict mode?

Reply:
`ListPrivateKeys()` can surely be used in strict mode (assuming Vault supports it). I see no reason why we should disallow it, as it just returns the key IDs, not the keys themselves.

Reply:
> as it just returns the key IDs, not the keys themselves.
ah, right